core/src/autogluon/core/hpo/executors.py

```diff
@@ -162,6 +162,7 @@ class RayHpoExecutor(HpoExecutor):
     }
     
     def __init__(self):
+        super().__init__()
```

Comment:
What is the impact of this?

Reply:
It's more like a bug fix. Previously, forgot to call `super().__init__()` will result in some property not being initialized in some case. It didn't cause problem as I set those property later. But added this just to be safe